src/main/cc/proxy_client/proxy_client.cc

```diff
@@ -104,7 +104,7 @@ string NormalizedRelativePath(const string& cwd, const string& path) {
   vector<string> segments = absl::StrSplit(rel_path, '/', absl::SkipEmpty());
   auto iter = segments.begin();
   while (iter != segments.end()) {
-    if (*iter == ".") {
+    if (*iter == "." && iter + 1 != segments.end()) {
```

Comment:
Why? What's wrong with a/. turning into a?

Reply:
I think this is turning "." into ""?

https://ideone.com/p7fqH6